packages/lesswrong/components/common/CloudinaryImage2.tsx

```diff
@@ -12,11 +12,12 @@ function cloudinaryPropsToStr(props) {
 // Cloudinary image without using cloudinary-react. Allows SSR. See:
 // https://github.com/LessWrong2/Lesswrong2/pull/937 "Drop cloudinary react"
 // https://github.com/LessWrong2/Lesswrong2/pull/964 "Temporarily revert removal of cloudinary-react"
-const CloudinaryImage2 = ({width, height, objectFit, publicId}: {
+const CloudinaryImage2 = ({width, height, objectFit, publicId, className}: {
```

Comment:
I actually ended up not using this component, but this seemed like a reasonable update to keep in here.